common/src/main/java/org/figuramc/figura/gui/screens/AbstractPanelScreen.java

```diff
@@ -216,7 +216,8 @@ public boolean keyPressed(int keyCode, int scanCode, int modifiers) {
         egg += (char) keyCode;
         egg = egg.substring(1);
         if (EGG.equals(egg)) {
-            Minecraft.getInstance().setScreen(new GameScreen(this));
+            Configs.DEBUG_MODE.setValue("true");
```

Comment:
Might as well make it a regular option on the screen, it doesn't need to be hidden behind a key combo. We already expose several options normal users don't mess with like the ip backend option.